spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -113,7 +113,12 @@ Similarly, the property `com.acme.ClassA/Retry/maxRetries` will be ignored if `@
 
 === Disabled Fault Tolerance policies with configuration
 
-Fault Tolerance policies can be disabled with configuration at method level, class level or globally for all deployment.
+Fault Tolerance policies can be disabled with configuration at method level, class level or globally for all deployment. If multiple configurations are specified, method-level configuration overrides class-level configuration, which then overrides global configuration. e.g.
+* `com.acme.test.MyClient/methodA/CircuitBreaker/enabled=true`
+* `com.acme.test.MyClient/CircuitBreaker/enabled=false`
```

Comment:
This middle line doesn't make a lot of sense. It's valid but has no effect because CircuitBreaker is disabled globally.

I think the example would be clearer if this line was removed.

Reply:
Alternatively, change the example to be global=false (as is), class-level=true, method-level=false and the explanation should state that all circuit breakers in the app are disabled, only those in the class except the method methodA are enabled.